src/app/forms/frontcountry-cabins/sections/frontcountry-cabins-section/frontcountry-cabins-section.component.html

```diff
@@ -9,6 +9,11 @@
           [type]="'number'"
         ></app-text-input>
         </div>
+        <app-info-text class="text-block"
+          [text]="
+            'Includes all adults, youth, and kids.'
+          "
```

Comment:
Not sure what's up with your formatting, but this could be one line. Same below.

Reply:
The formatting was from an another section that was nested enough that it needed to use this format to keep lines shorter. Not necessary here. Updating PR.